app/controllers/users_controller.rb

```diff
@@ -37,14 +35,22 @@ def create
       end
     end
 
-    redirect_to project_users_url(@project)
+    respond_to do |format|
+      format.js { render :refresh_user_list }
+      format.html { redirect_to project_users_url(@project) }
+    end
   end
 
   def destroy
     @project = current_user.projects.friendly.find(params[:project_id])
     @user = @project.users.find(params[:id])
     @project.users.delete(@user)
-    redirect_to project_users_url(@project)
+    flash[:notice] = "#{@user.email} was removed from this project"
+
+    respond_to do |format|
+      format.js { render :refresh_user_list }
```

Comment:
puts verdade, se for manter no js, vai ficar mais complexo, deixa assim por enquanto depois a gente vai acertando